modular_stonehedge/licensed-death-rattler/Death-Rattler/code/Death-Rattler.dm

```diff
@@ -0,0 +1,158 @@
+// This code is part of Death-Rattler
+// Copyright (C) 2024 Moribund/Luctus-Comptus
+// Unauthorized use, modification, or distribution of this code is prohibited without copyright holders approval. See LICENSE.md
```

Comment:
```suggestion
```
not allowed under section 7 and section 10 of the AGPLv3.
section 7:
> All other non-permissive additional terms are considered "further restrictions" within the meaning of section 10. If the Program as you received it, or any part of it, contains a notice stating that it is governed by this License along with a term that is a further restriction, you may remove that term. If a license document contains a further restriction but permits relicensing or conveying under this License, you may add to a covered work material governed by the terms of that license document, provided that the further restriction does not survive such relicensing or conveying. 

section 10:
> You may not impose any further restrictions on the exercise of the rights granted or affirmed under this License. For example, you may not impose a license fee, royalty, or other charge for exercise of rights granted under this License, and you may not initiate litigation (including a cross-claim or counterclaim in a lawsuit) alleging that any patent claim is infringed by making, using, selling, offering for sale, or importing the Program or any portion of it.